### DISCUSSION TITLE
Usage of „free_license“ stanza

### DISCUSSION THREAD

**Discussion by stonerl** [Homebrew/discussions#4712] [Writing Formulae/Casks]
Should this stanza also be added to all Casks that download a free trial?

I installed some Casks that downloaded commercial software which allows a trial period. But I can't recall ever seeing a warning message.

So I would add this stanza to some casks but want to make sure that this is correct before starting.

**Comment by p-linnane** [comment]
This is more geared toward "you need a license key to use this, but you can get it for free at `<url>`". It doesn't pertain to free trials.
